libcob/common.h

```diff
@@ -1408,6 +1408,8 @@ typedef struct __cob_module {
 	unsigned char		flag_unused[6];	/* For future, use for new flags etc */
 
 	unsigned char		unused[32];		/* For future use */
+
+	const char	*gc_version;	/* module version, until 3.1.2: set by cob_check_version */
```

Comment:
just to ask for _some_ change: please drop the `, until 3.1.2: set by cob_check_version` part (GC4 has no binary compat to any previous version and shouldn't)

Reply:
Done.